docs/products/m3db/index.rst

```diff
@@ -1,18 +1,27 @@
-M3DB Welcome
+Aiven for M3
 ============
 
-M3DB is an open source, **distributed time series database**. It's ideal for organisations that have a very large volume of metrics to handle. This platform was originally developed at Uber, so that gives an idea of the scale of metrics that it can handle.
+Aiven for M3 is a fully managed **distributed time series database**, deployable in the cloud of your choice which can bring unlimited scalability and high-availability to your Prometheus monitoring environment and other time series applications.
 
-M3DB plays nicely with other tools that might be in the stack already:
+Why M3?
+-------
+M3 is a great choice if your organization has a very large volume of metrics to handle, and it can be used as part of your observability solution. 
 
-* Collect metrics with Prometheus; M3DB is designed as a scalable storage backend for Prometheus.
-* Write metrics direct from your applications using the InfluxDB-compatible wire protocol.
-* Dashboards and query features available using Grafana and PromQL.
+M3 is highly compatible with other Aiven products for the following tasks:
+
+* To collect metrics with Prometheus: M3DB is designed as a scalable storage backend for Prometheus.
+* To write metrics directly from your applications using the InfluxDB-compatible wire protocol.
+* To create dashboards and query available features using Grafana and PromQL.
+
+Check out all the features on our `M3 product page <https://aiven.io/m3#full-feature-list/>`_. 
+
+.. tip::
+   Aiven offers M3 because we ourselves needed a solution that would work with the size of our own metrics - and we love it!
 
 First Steps
 -----------
 
-Check out our :doc:`getting-started` guide for M3DB and take your first steps right away.
+Check out our :doc:`getting-started` guide for M3 and take your first steps right away.
```

Comment:
This made me realise that my use of M3 vs M3DB is all over the place! M3 is the overall project, M3DB is the actual database. There's a bit more on the components on their own docs: https://m3db.io/docs/overview/components/ which might or might not help work out what to do here.

Reply:
I think we should specify what the main product name actually is, and in which cases should we use what. I see M3 and M3DB used interchangeably.